### PR TITLE
MPP-2087 - Strings for new announcement item - EU Country Expansion

### PR DESCRIPTION
**Pull request by lloan**
Strings for MPP-2087: https://mozilla-hub.atlassian.net/browse/MPP-2087 

image

image

### CODE REVIEW

File: en/whatsnew.ftl [hunk 1]
```diff
@@ -87,3 +87,10 @@ whatsnew-feature-mailing-list-heading = Get help protecting your privacy
 whatsnew-feature-mailing-list-snippet = It’s not easy to keep up with the latest privacy and security tips…
 whatsnew-feature-mailing-list-description = It’s not easy to keep up with the latest privacy and security tips and habits. So we started a newsletter to help you make your online life more secure.
 whatsnew-feature-mailing-list-cta = Subscribe to the newsletter
+
+whatsnew-feature-premium-expansion-eu-heading = Unlimited masks now available
+# A preview of the full content of `whatsnew-feature-premium-expansion-eu-description`.
+# Try to keep this string short, not longer than the English version.
+whatsnew-feature-premium-expansion-eu-snippet = { -brand-name-firefox } { -brand-name-relay-premium } is now available in your country…
```
Comment (flodolo) [inline en/whatsnew.ftl:94]:
```suggestion
whatsnew-feature-premium-expansion-eu-snippet = { -brand-name-firefox-relay-premium } is now available in your country…
```

There is a term for that, you should never concatenate different terms.

File: en/whatsnew.ftl [hunk 2]
```diff
@@ -87,3 +87,10 @@ whatsnew-feature-mailing-list-heading = Get help protecting your privacy
...
+whatsnew-feature-premium-expansion-eu-description = { -brand-name-firefox } { -brand-name-relay-premium } is now available in your country. Protect your inbox with unlimited email masks, and get a unique { -brand-name-relay } email domain to make creating masks on the go easy and secure.
```
Comment (flodolo) [inline en/whatsnew.ftl:95]:
```suggestion
whatsnew-feature-premium-expansion-eu-description = { -brand-name-firefox-relay-premium } is now available in your country. Protect your inbox with unlimited email masks, and get a unique { -brand-name-relay } email domain to make creating masks on the go easy and secure.
```

**Review by flodolo — CHANGES_REQUESTED** [review]
Are there other strings in the work, or can I land this once the term issues are resolved?

**Comment by lloan** [comment]
> Are there other strings in the work, or can I land this once the term issues are resolved?

These are only strings for this feature. This can be landed any time. Thanks!

**Review by flodolo — APPROVED** [review]

**Pull request merged**